engine/apps/schedules/migrations/0012_auto_20230502_1259.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.18 on 2023-05-02 12:59
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('schedules', '0011_oncallschedule_cached_ical_final_schedule'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
```

Comment:
Looks like nothing changed about the model manager, is this expected?

Reply:
This was added by the automatic migration generation, will double-check.

Reply:
This seems related to a previous change in the manager, for which there wasn't a migration created. I think the change was adding a method to the custom base queryset, which is used as the default `objects` manager via `PolymorphicManager` (so, not a real model/db change, but it seems this is detected as a modification for which a migration step is added).